x-pack/plugins/apm/server/lib/transaction_groups/get_error_rate.ts

```diff
@@ -42,25 +43,33 @@ export async function getErrorRate({
   const filter = [
     { term: { [SERVICE_NAME]: serviceName } },
     { range: rangeFilter(start, end) },
-    { exists: { field: HTTP_RESPONSE_STATUS_CODE } },
+    { exists: { field: EVENT_OUTCOME } },
     ...transactionNamefilter,
     ...transactionTypefilter,
     ...uiFiltersES,
   ];
 
+  const mustNot = [
+    {
+      term: {
+        [EVENT_OUTCOME]: EventOutcome.unknown,
+      },
+    },
+  ];
```

Comment:
Can you use `{ terms: { [EVENT_OUTCOME]: [ EventOutcome.success, EventOutcome.failure ] } }` instead of the exists query + must_not clause?